adapters/pubmatic/pubmatic_test.go

```diff
@@ -716,3 +787,132 @@ func TestGetMapFromJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestGetDisplayManagerAndVer(t *testing.T) {
+	type args struct {
+		app *openrtb2.App
+	}
+	type want struct {
+		displayManager    string
+		displayManagerVer string
+	}
+	tests := []struct {
+		name string
+		args args
+		want want
+	}{
+		{
+			name: "request app object is not nil but app.ext has no source and version",
```

Comment:
Please update the `name` strings throughout this test substituting dashes or underscores for spaces. `t.Run` does this automatically so doing this in the tests makes it easy to search for failed tests in the code base.
e.g. `name: "request-app-object-is-not-nil-but-app.ext-has-no-source-and-version"`

Reply:
Done